compiler/rustc_mir/src/transform/check_consts/ops.rs

```diff
@@ -238,13 +238,32 @@ impl NonConstOp for TransientCellBorrow {
 pub struct CellBorrow;
 impl NonConstOp for CellBorrow {
     fn build_error(&self, ccx: &ConstCx<'_, 'tcx>, span: Span) -> DiagnosticBuilder<'tcx> {
-        struct_span_err!(
+        let mut err = struct_span_err!(
             ccx.tcx.sess,
             span,
             E0492,
-            "this borrow to an interior mutable value may end up in the final value of this {}",
+            "{}s cannot refer to interior mutable data",
             ccx.const_kind(),
-        )
+        );
+        err.span_label(
+            span,
+            format!("this borrow of an interior mutable value may end up in the final value"),
+        );
+        if let hir::ConstContext::Static(_) = ccx.const_kind() {
+            err.help(
+                "To fix this, the value can be extracted to separate \
+                `static` and then referenced.",
```

Comment:
Don't we usually use lower-case and no trailing `.`?

Reply:
oh... yea that was my mistake due to the `teach` messages actually starting with capitalization, and I moved this around a bit.

Reply:
fixed